crates/nu-command/Cargo.toml

```diff
@@ -24,6 +24,7 @@ nu-table = { version = "0.27.2", path = "../nu-table" }
 nu-test-support = { version = "0.27.2", path = "../nu-test-support" }
 nu-value-ext = { version = "0.27.2", path = "../nu-value-ext" }
 nu-ansi-term = { version = "0.27.2", path = "../nu-ansi-term" }
+nu-platform-specifics = { version = "0.27.2", path = "../nu-platform-specifics" }
```

Comment:
Not sure if this needs its own crate. Can the platform specific stuff be provided by the host?  For the commandline nushell, I was wondering if maybe nu-cli should handle printing the errors and how `where` works.

Reply:
I've removed nu-platform-specifics. Currently we only have one BasicHost implementation, so I moved the functionality over there.

I think of nu-cli as a crate that provides a commandline-editor and not much more. I don't think that printing of errors should be done there, nor how where works. Or am I getting something wrong here? But I am happy to hear your pov here :)